php/examples/users.php

```diff
@@ -20,58 +20,52 @@
 /**
  * @param array $installedUsers
  *
```

Comment:
Why is this int to begin with? The PBX has login IDS as strings, e.g. 0001

Reply:
interesting! The code works with intval() conversions and starts counting from 1000. 
So now that you mention it... this already looks like a big workaround o.O

Reply:
Yes agreed. Maybe the original author was unaware of the difference between login ID and account ID, the latter of which are integers beginning from 1000. These cannot and must not be set or changed.

Reply:
Or the original author wanted to keep it as simple as possible and not do any zero-fill string shenanigans in an example implementation.